### ISSUE TITLE
Articles - Comments only display on Index view, but not on Display view

### ISSUE THREAD

**Issue by DeadSuperHero** [redmatrix/hubzilla#1013]
Here's an example of what's happening:

My "Suggestions for Leading a Happier Life" article has a handful of federated comments on it: https://deadsuperhero.com/articles/dsh/

However, on the display view of the article, none of the comments appear: https://deadsuperhero.com/articles/dsh/suggestions-happier-life

What would be great is if there were a way to invert this logic, so that the index view basically acts like Hubzilla blog mode for posts, and the Display is where the comments can be read.

**Comment by DeadSuperHero** [comment]
This also appears to happen to other interactions such as likes, not just comments.

**Comment by zotlabs** [comment]
Found the bug. Should land in Hubzilla (dev) in a day or two. Haven't done anything about inverting the logic.

**Comment by DeadSuperHero** [comment]
> Haven't done anything about inverting the logic.

Fair enough, there are probably other methods of approaching this anyway.
